cmd/utils/flags.go

```diff
@@ -461,6 +461,10 @@ var (
 		Name:  "rpc.gascap",
 		Usage: "Sets a cap on gas that can be used in klay_call/estimateGas",
 	}
+	RPCGlobalTxFeeCapFlag = cli.Float64Flag{
```

Comment:
Do we need to support this flag? 
I don't think adding this flag only for ethereum api is necessary
If you think we have to add, then change the flag name for more specific for eth api better.

Reply:
@jimni1222 
I changed the flag name.
There are some logic to use this RPCTxFeeCap in Eth namespace APIs, so I'd like to keep this flag.